modules/reference/pages/data-transforms/js/index.adoc

```diff
@@ -0,0 +1,2 @@
+= JavaScript SDK for Data Transforms
```

Comment:
should index page be JavaScript SKDs (plural)?

Reply:
also, needs description

Reply:
The SDK includes API packages